library/src/main/java/io/ona/kujaku/comparisons/EqualToComparison.java

```diff
@@ -0,0 +1,24 @@
+package io.ona.kujaku.comparisons;
+
+
+public class EqualToComparison extends Comparison {
+    private static final String TAG = "EqualToComparison";
+
+    @Override
+    public boolean compare(String a, String type, String b) {
+        switch (type) {
+            case TYPE_STRING:
```

Comment:
Why does type need to be tested yet the arguments to be compared will always be type `String`?

Reply:
> Actually according to the spec [here](https://github.com/onaio/kujaku/issues/237#issue-396650138), you might need to add a toggle button anyway.

Cool, implemented according to the spec which says

> * Add a button to the GeoWidget that draws the arrows. No animation is needed, we just need arrows to show up on the map